.github/workflows/sonarcloud.yml

```diff
@@ -9,14 +9,17 @@ jobs:
   sonarcloud:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '11'
-
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
+      - name: set-up-environment
+        uses: DFE-Digital/github-actions/set-up-environment@master
```

Comment:
I'm somewhat wary of the `@master` - one bad commit to that repo will break builds for every dependant project in DFE?

Would it not be better to use tags and make the projects choose the versions they use? That being said we're using `@master` for the slack notifier and again, a bad change will break all our repos so maybe its not really a problem?

Reply:
its our team that manages this via PRs and we are reviewing and testing, so Master should be protected